### PR TITLE
Feature: Label and description for Inputs

### PR DESCRIPTION
**Pull request by csansoon**
Autoimported `Input` components now have optional `label` and `description` props.

### CODE REVIEW

File: apps/server/src/lib/autoimports/forms/input/index.svelte [hunk 1]
```diff
@@ -25,10 +35,18 @@
...
+  {/if}
+  <Input
+    on:input={handleInput}
+    value={$inputStore}
+    type={type}
+    checked={type === "checkbox" && !!$inputStore}
+    {...$$restProps}
+  />
+  {#if description}
+    <p class="text-sm text-muted-foreground">{description}</p>
```
Comment (andresgutgon) [inline apps/server/src/lib/autoimports/forms/input/index.svelte:50]: Use Text component

File: packages/client/svelte/src/lib/ui/label/label.svelte [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  let className: $$Props["class"] = undefined;
+  export { className as class };
+</script>
+
+<LabelPrimitive.Root
+  class={theme.ui.label.cssClass({ className })}
+  {...$$restProps}
+>
+  <slot />
```
Comment (andresgutgon) [inline packages/client/svelte/src/lib/ui/label/label.svelte:15]: Use our `<Text />` element here to control how text is render also use `asChild` prop from bits https://www.bits-ui.com/docs/components/label

File: packages/client/core/src/theme/ui/label/index.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { cn } from '../../utils'
+
+export function cssClass({ className }: { className?: string | null }) {
+  return cn('text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70', className)
```
Comment (andresgutgon) [inline packages/client/core/src/theme/ui/label/index.ts:4]: Use text component in

File: packages/client/svelte/src/lib/ui/label/label.svelte (hunk 2 above)
Comment (csansoon) [reply packages/client/svelte/src/lib/ui/label/label.svelte]: Not sure I understand why to use `asChild` here

**Review by geclos — DISMISSED** [review]

File: packages/client/svelte/src/lib/ui/label/index.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+import Root from "./label.svelte";
+export default Root;
```
[the remark this answers is not in the document]
Comment (geclos) [reply packages/client/svelte/src/lib/ui/label/index.ts]: yeah it was a mistake on our part when we start this project honestly, we will gradually migrate away from this unnecessary pattern in other places

**Review by andresgutgon — APPROVED** [review]

**Pull request merged**